woocommerce-services.php

```diff
@@ -208,6 +207,17 @@ public static function plugin_deactivation() {
 
 		public static function plugin_uninstall() {
 			WC_Connect_Options::delete_all_options();
+			self::delete_notices();
```

Comment:
I thought about cleaning the note also on plugin_deactivation but that means on upgrades, the note would be displayed again because [the plugin is deactivated](https://core.trac.wordpress.org/browser/tags/5.7/src/wp-admin/includes/class-plugin-upgrader.php#L512).